Protection Profile/BiocPP.adoc

```diff
@@ -452,14 +452,25 @@ Target error rates defined in SFR shall be evaluated based on <<BIOSD>>. Normall
 ==== Protection of the TSF (FPT)
 ===== FPT_BDP_EXT.1 Biometric data processing [[FPT_BDP_EXT.1]]
 
-*FPT_BDP_EXT.1.1*:: Processing of plaintext biometric data used to generate templates and perform sample comparisons shall be hardware-isolated from the OS on the TSF in runtime.
+*FPT_BDP_EXT.1.1*:: Processing of plaintext biometric data used to generate templates and perform sample comparisons shall be hardware-isolated from the main computer operating system on the TSF in runtime.
```

Comment:
Sorry, since I have been "absent" for a while and have missed any discussions..
Could we consider "platform operating system" rather than "main computer operating system"? It might be a clearer term, especially for MDF PP and GPOS PP.

Reply:
@The-Fiona so it could, but it happened to be that "main computer operating system" was how we defined the difference between say Android and the TEE (or iOS and the Security Enclave) when we had to redefine the SEE for the integration to MDF. We could make that change though, but it would have to be done in a lot of places, not just here.